CONFIG.example

```diff
@@ -13,7 +13,7 @@ VIVADO_VER = 2022.2
 # Definitions needed for FPGA build
 export VIVADO = /scratch/Xilinx/Vivado/\$(VIVADO_VER)/settings64.sh
 # Path to external IP repository (if required)
-#export EXT_IP_REPO =
+export EXT_IP_REPO = /scratch/Xilinx/XAPP589/PICXO_FRACXO_v32_2
```

Comment:
Could we have this IP available in a shared location under `/dls_sw/...`

Reply:
We need to make sure this is installed on the S3 bucket too...

Reply:
Yes, this is the path within the container. I will add to /dls_sw too..